pkgs/by-name/go/golem/package.nix

```diff
@@ -40,16 +38,10 @@ rustPlatform.buildRustPackage rec {
     rustPlatform.bindgenHook
   ];
 
-  buildInputs =
-    [
-      fontconfig
-      (lib.getDev openssl)
-    ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [
```

Comment:
I think this is/was needed because of build issues on x86_64-darwin. cc @GaetanLepage for more details

Reply:
That PR landed before https://github.com/NixOS/nixpkgs/pull/352129 hit `master` a few days ago. It shouldn’t be necessary any more (unless this actually needs functionality from the macOS 15 SDK, but I think that’s unlikely).